content/documentation/3-developer-guides/3-batch/1-simple-task.md

```diff
@@ -11,31 +11,293 @@ In this guide we will develop a batch application using Spring Cloud Task and de
 We will start from initializr and create two Spring Cloud Batch applications.
 Note for CF we need a manifest, for k8s we need a service/deployment yaml.
 
+NOTE: All code for this project can be found here: https://github.com/cppwfs/edutasksamples
+
 ## Development
 
-Explaine what we are doing to do development wise.
+Imagine you are working for a cell phone data provider and you are responsible for writing the solution that will create the billing statements for the customers. The usage data that you will be using will be stored in JSON files stored in a file on your file system. Your solution will be responsible for pulling the data from these files, storing the raw data into a table in your database and then generate a price from this usage data and storing it into a billing_statement table.  
+We could implement this entire solution into a single Spring Boot Application that utilizes Spring Batch, however for this example we will break up the solution into 2 phases:
+
+1. BillUsage will be a Spring Boot application using Spring Cloud Task that will read data from JSON files from the classpath and put the data into the `BILL_USAGE` table.
```

Comment:
No. Application names do not get formatted as code. That can become a very fine distinction for a microservice or other application that itself gets called by other code. The rule is that, if a developer has to write it as code in the current context (that is, in a code sample) it gets code formatting. Otherwise, it's treated as a product name and not put in code format.